packages/node_modules/@webex/plugin-meetings/sample/app.js

```diff
@@ -8,6 +8,8 @@
 /* eslint-disable no-console */
 /* eslint-disable require-jsdoc */
 
+import '@ciscospark/internal-plugin-calendar';
```

Comment:
Does this work? I didn't think we could import anything inside of `/sample/` but maybe that was the old way

Reply:
it seems to work, yeah i tested it

Reply:
ok. For consistency, I think we should move it to ciscospark.js where we require other plugins. Isn't that what @arun3528 did for device search?